### PR TITLE
improve handling windows update

### PR DESCRIPTION
**Pull request by mszhanyi**
fix https://github.com/pytorch/pytorch/issues/64536

1. Add `sc stop wuauserv` as well
2. Restore windows update after cuda installation.
3. Improve readability of the log output.

testing PR https://github.com/pytorch/pytorch/pull/63551

### CODE REVIEW

**Comment by mszhanyi** [comment]
@peterjc123 @malfet @seemethere

**Comment by malfet** [comment]
@mszhanyi please do not re-enable wsupdate( as it's likely slow down test), otherwise looks good to me.

**Comment by mszhanyi** [comment]
> @mszhanyi please do not re-enable wsupdate( as it's likely slow down test), otherwise looks good to me.

@malfet Could we re-enable them for some time?
I'm a little afraid that the reason why circleci machines start wsupdate frequently AGAIN in this week is that we have disabled the wsupdate for a long time. From Aug20 to Aug27, there isn't any errors caused by the wsupdate. Maybe we need to give these machine an chance to update.

I think what we might to do is to ensure no wsupdate when installing cuda.

**Review by seemethere — APPROVED** [review]

**Comment by seemethere** [comment]
> > @mszhanyi please do not re-enable wsupdate( as it's likely slow down test), otherwise looks good to me.
> 
> @malfet Could we re-enable them for some time?
> I'm a little afraid that the reason why circleci machines start wsupdate frequently AGAIN in this week is that we have disabled the wsupdate for a long time. So disabling the service become invalid. From Aug20 to Aug27, there isn't any errors caused by the wsupdate. Maybe we need to give these machine an chance to update.
> 
> I think what we might to do is to ensure no wsupdate when installing cuda.

We shouldn't be afraid of this, the CircleCI machines shouldn't be long lived so we shouldn't worry about turning updates off on them

File: windows/internal/cuda_install.bat [hunk 1]
```diff
@@ -232,3 +237,7 @@ echo Installing GPU driver DLLs
 
 echo Cleaning temp files
 rd /s /q "%SRC_DIR%\temp_build" || ver > nul
+
+sc config wuauserv start= auto
+sc start wuauserv
+sc query wuauserv
```
Comment (malfet) [inline windows/internal/cuda_install.bat:243]: Let's not re-enable it on a short lived instances
```suggestion
```

**Pull request merged**